composer.json

```diff
@@ -17,7 +17,7 @@
     "homepage": "https://github.com/erickskrauch/php-cs-fixer-custom-fixers",
     "require": {
         "php": "^7.4 || ^8.0",
-        "friendsofphp/php-cs-fixer": "^3"
+        "friendsofphp/php-cs-fixer": "^3.59"
```

Comment:
Seems like the `ConfigurableFixerTrait` has been introduced only on version `3.59.3`. So you have to be more strict about it.